tests/unit/email-dropdown.spec.js

```diff
@@ -112,6 +112,21 @@ describe("EmailDropdown.vue", () => {
       expect(wrapper.emitted().input[1]).to.have.length(1);
       expect(wrapper.emitted().input[1][0]).to.be.equal("hello@gmail.");
     });
+
+    it("returns 'true' if input is empty after clicking clean (X) button", async () => {
```

Comment:
In this case, the test title y a little bit blur and the intention is not clear. We can't say 'this returns true...' since it is the test that is returning true instead of the code per se.
```suggestion
    it("clears the email field when the x is clicked", async () => {
```